### PR TITLE
Let Listener connect to TLS-enabled Stomp services

### PR DESCRIPTION
**Pull request by mjgiarlo**
Refs LD4P/sinopia_indexing_pipeline#18

Use a new config value to turn this on. Default is off. New environment variable is `BROKER_TLS_ENABLED`.

### CODE REVIEW

**Comment by mjgiarlo** [comment]
Cc: @rsmith11

File: src/Listener.js [hunk 1]
```diff
@@ -4,8 +4,15 @@ import Logger from './Logger'
 
 export default class Listener {
   constructor() {
+    const stompOptions = {
+      host: Config.brokerHost,
+      post: Config.brokerPort
```
Comment (jmartin-sul) [inline src/Listener.js]: typo?  `post` -> `port`?

Comment (mjgiarlo) [reply src/Listener.js]: great catch! pushed.

**Comment by mjgiarlo** [comment]
@jmartin-sul ready for another look

**Pull request merged**